### PR TITLE
Chapter - Web App Security Review

### PR DESCRIPTION
**Pull request by jdotpz**

### CODE REVIEW

File: webapp-security-review.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+* Verify that nested archives are rejected
+
+## Error handling
+* Verify that error pages (e.g. 404 and 500) do not contain sensitive information (path, system status or setup * information, debug messages)
+* Verify that global debug mode is disabled
+* Verify that user input in error messages is correctly encoded (XSS) and newlines are not written to text files as is.
+
+
+## Recommended Reading & References
+[The Tangled Web: A Guide to Securing Web Applications](https://anawesomelink.com)
```
Comment (simonwex) [inline webapp-security-review.md]: What's your plan for this?

Comment (jdotpz) [inline webapp-security-review.md]: My thought is that we can link to resources to indicate how to test for each item in the list as a reference.  Perhaps then we can provide a template checklist as well in a git issue that people can copy/paste into their own issue for review.

I think I'll probably need a bunch of help from @toolness to get the list populated with good resources or tutorials.  What do you think Atul?

Comment (toolness) [inline webapp-security-review.md]: Sure, happy to help!

**Comment by Pomax** [comment]
time passed this PR by =(

**Pull request closed without merging**